### ISSUE TITLE
 Dynamic annotations

### ISSUE THREAD

**Issue by KateNegrienko** [ihor-zinchenko/ngx-apple-maps#11]
Hi, 

I tried to change annotations after loading. Adding works fine, but when I tried delete one annotation see error:

```
ERROR TypeError: Cannot read property 'removeAnnotation' of undefined
    at AppleMapsAnnotationComponent.ngOnDestroy (ngx-apple-maps.js:1265)
```

![image](https://user-images.githubusercontent.com/13833386/105627352-2a444580-5e3f-11eb-980a-b46e3216f75a.png)

I tried to find this case in documentation, but I don't see any use cases like this
How I can remove Annotation? 

Thanks

**Comment by ihor-zinchenko** [comment]
@KateNegrienko hi! ill try to reproduce that and back to you shortly!

**Comment by KateNegrienko** [comment]
Thanks

**Comment by leonardopalma** [comment]
Has a solution been found?
I've the same problems.

Thanks

**Comment by ihor-zinchenko** [comment]
Fixed in v1.0.0. Older versions are no longer supported. It will be released soon.
